src/core_modules/capture-core/rules/converters/inputConverter.js

```diff
@@ -6,117 +6,56 @@ import type { IConvertInputRulesValue } from 'capture-core-utils/rulesEngine/rul
 
 const dateMomentFormat = 'YYYY-MM-DD';
 
-class RulesValueConverter implements IConvertInputRulesValue {
-    convertText(value: any): string {
-        return value || '';
-    }
-
-    convertLongText(value: any): string {
-        return value || '';
-    }
-
-    convertLetter(value: any): string {
-        return value || '';
-    }
-
-    convertPhoneNumber(value: any): string {
-        return value || '';
-    }
-
-    convertEmail(value: any): string {
-        return value || '';
-    }
-
-    convertBoolean(value: any): boolean | string {
-        return (value || value === false) ? value : '';
-    }
-
-    convertTrueOnly(value: any): boolean | string {
-        return (value || value === false) ? value : '';
-    }
-
-    convertDate(value: any): string {
+const convertStringValue = (value: ?string): ?string => (value !== '' && value || null);
```

Comment:
`value !== '' && value || null` can be replaced by `value || null` if I'm not missing something

Reply:
Ahaha.. :sweat_smile:  You're right